packages/adapters/solana/src/client.ts

```diff
@@ -360,37 +352,21 @@ export class SolanaAdapter implements ChainAdapter {
   }
 
   // -- Private -----------------------------------------------------------------
-  private async syncAccount({
-    address,
-    caipNetwork
-  }: {
-    address: string | undefined
-    caipNetwork?: CaipNetwork | undefined
-  }) {
-    const currentCaipNetwork = caipNetwork || this.appKit?.getCaipNetwork()
-    const solanaNetwork =
-      currentCaipNetwork?.chainNamespace === CommonConstantsUtil.CHAIN.SOLANA
-        ? currentCaipNetwork
-        : this.appKit?.getCaipNetwork(this.chainNamespace)
```

Comment:
For some reason this logic here was getting a CaipNetwork with a RPC url that did not contain the project id. Only getting the caip network from appkit should do the trick.